src/settings/LocationCampuses.js

```diff
@@ -84,23 +91,43 @@ class LocationCampuses extends React.Component {
   componentDidMount() {
     const institutionId = sessionStorage.getItem('institutionIdCampuses');
     this.setState({ institutionId });
-    ['institutions', 'locationsPerCampus'].forEach(i => {
+    ['institutions', 'libraries'].forEach(i => {
       this.props.mutator[i].reset();
       this.props.mutator[i].GET();
     });
   }
 
   numberOfObjectsFormatter = (item) => {
-    const records = (this.props.resources.locationsPerCampus || {}).records || [];
-    return records.reduce((count, loc) => {
+    const records = (this.props.resources.libraries || {}).records || [];
+    const numberOfObjects = records.reduce((count, loc) => {
       return loc.campusId === item.id ? count + 1 : count;
     }, 0);
+
+    const onNumberOfObjectsClick = () => {
+      this.props.history.replace({
+        pathname: 'location-libraries'
+      });
+
+      sessionStorage.setItem('institutionIdLibraries', this.state.institutionId);
+      sessionStorage.setItem('campusIdLibraries', item.id);
+    };
+
+    return (
+    // eslint-disable-next-line jsx-a11y/click-events-have-key-events,jsx-a11y/no-static-element-interactions
```

Comment:
it's invalid, our application should be accessible

or you make div accessible, or you use button

Reply:
in fact it should be a link, not a button because your callback changes pathname

Reply:
I agree this has to be a <Link> component

Reply:
Thanks @NikitaSedyx. I used `TextLink` in that case, and also made changes in previous place.